docs/FilePropertyManagement.md

```diff
@@ -132,34 +132,34 @@ You can view a sample individual artifact properties file, [epsmlist.cbl.propert
 
-This approach requires:
+This approach consists of:
 
-- a mapping of the application artifact(s) to a language definition
-- a property file for defining the build parameters for each language definition
+- Language definition mapping file: A mapping of the application artifact(s) to a language definition
+- Language definition properties file(s): For each language definition, a properties file defining that language definition's build parameters
 
-The Language Definition Property approach can be enabled by setting the property `loadLanguageDefinitionProperties` in the `application-conf/application.properties` file to `true`. To enable this option for a specific file or a set of files, use the DBB file property syntax and set  `loadLanguageDefinitionProperties` to `true` in the `application-conf/file.properties` file. Below is a sample to enable Language Definition Mapping for all programs starting with `eps` and `lga` via the `application-conf/file.properties` file:
+The "language definition mapping" approach can be enabled by setting the property `loadLanguageDefinitionProperties` in the `application-conf/application.properties` file to `true`. To enable this option for a specific file or a set of files, use the DBB file property syntax and set  `loadLanguageDefinitionProperties` to `true` in the `application-conf/file.properties` file. Below is a sample to enable language definition mapping for all programs starting with `eps` and `lga` via the `application-conf/file.properties` file:
 
 ```properties
 loadLanguageDefinitionProperties = true :: **/cobol/eps*.cbl, **/cobol/lga*.cbl
 ```
 
-The Language Definition Property files need to be created in the `build-conf` folder. This will allow you to define file properties that support overriding in the Language Definition Property file. You can create multiple Language Definition Property files under `build-conf` folder to serve different variations or types. A sample Language Definition Property file can be found at [langDefProps01.properties](../build-conf/langDefProps01.properties).  
+Build properties for a language definition are specified in a language definition properties file, which should be created in the `build-conf` folder. This will allow you to define file properties that support overriding in the language definition properties file. You can implement multiple language definitions to serve different variations or types by creating multiple language definition properties files under the `build-conf` folder. A sample language definition properties file can be found at [langDefProps01.properties](../build-conf/langDefProps01.properties).  
```

Comment:
I don't understand this sentence ... 
Should we just remove it?

```
This will allow you to define file properties that support overriding in the language definition properties file
```

Reply:
You found the one sentence I was still thinking about how to reword. 😆 I also had considered removing it as long as no meaning was lost.